Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "watchexec"
-version = "1.9.2"
+version = "1.9.3"
```

```suggestion
version = "1.9.2"
```

Please do not bump the version in PR :) I'll do that on release